flow/scripts/synth.tcl

```diff
@@ -65,8 +65,10 @@ foreach cell $::env(DONT_USE_CELLS) {
 # Technology mapping of flip-flops
 # dfflibmap only supports one liberty file
 if {[env_var_exists_and_non_empty DFF_LIB_FILE]} {
+  clockgate -liberty $::env(DFF_LIB_FILE) -min_net_size 8 {*}$dfflibmap_args
```

Comment:
The `min_net_size` value could be a flow parameter, do you want me to make it one? `MIN_CLOCK_GATE_FF_GROUP` ?

Reply:
Yes